src/main/java/org/cryptomator/cli/Args.java

```diff
@@ -162,4 +173,14 @@ public Path getFuseMountPoint(String vaultName) {
 		Path mountPointPath = Paths.get(mountPoint);
 		return mountPointPath;
 	}
+
+	public boolean hasVersion() {
+		return hasVersion;
+	}
+
+	public void printVersionAndExit() {
+		String version = Version.IMPLEMENTATION_VERSION;
+		System.out.println(version);
+		System.exit(0);
```

Comment:
Is it easily possible to return from the `main()` function instead of calling `System.exit`? I would prefer this, but am fine with `System.exit` if it is _a lot_ simpler.

Reply:
Yes! This is a good suggestion. Initially I did not want to change the main() method because it looks very clean and straightforward. However at the same time, using `System.exit` as well as having the print and exit logic in `Args.java` is less than ideal, it should be in the main class.